### DISCUSSION TITLE
Bypass the SELECT DEVICE in the on-screen wizard?  and RESUSCITATE guide help?

### DISCUSSION THREAD

**Discussion by GR0DZ1LLA** [GerryDazoo/Slinger#606] [Q&A]
I have two SB500's and I don't have the passwords.

I factory reset both, and upgraded the firmware from 1.3.612 to 1.8.402, because it was the only way to move forward using the on-screen wizard.

The issue is that, in the on-screen wizard, when you have to pick a DEVICE (cablebox, etc), it polls a server and fails and I can't choose any device and keeps me from advancing passed it.

I see the ACCOUNT portion of the wizard but cannot get to it because of picking a device stalemate.

I was hoping that by putting it "in my account" that I could use SlingPlayer v5.0.0.83 and get the logs?

I'm kind of lost here.  What am I supposed to do at this point?

I can view the console.log for SlingPlayer and extract the hashed password.

I try to use the resuscitate guide, but get stuck after making a new device and trying to select SLINGBOX SETTINGS.  It never gets passed step 8.

Gerry says that Slinger will process hashed passwords and v4 of Slinger will also provide the password in plain-text.

But I cannot get the device passed the wizard or registered to my Slingbox account.

Any advice?

**Comment by GerryDazoo** [comment]
You have to install a compatible "custom remote" file. This doesn't require the servers.

**Reply by GR0DZ1LLA** [reply]
Ok, is there any documentation regarding this?  I don't know what that means, when I see "custom remote" I think about a handheld remote for a TV.  Then I think about the webUI-based remote.  I think both of these are wrong, so clearly I don't know what this means.

**Comment by lead2crm** [comment]
> I have two SB500's and I don't have the passwords.
> 
> I factory reset both, and upgraded the firmware from 1.3.612 to 1.8.402, because it was the only way to move forward using the on-screen wizard.
> 
> The issue is that, in the on-screen wizard, when you have to pick a DEVICE (cablebox, etc), it polls a server and fails and I can't choose any device and keeps me from advancing passed it.
> 
> I see the ACCOUNT portion of the wizard but cannot get to it because of picking a device stalemate.
> 
> I was hoping that by putting it "in my account" that I could use SlingPlayer v5.0.0.83 and get the logs?
> 
> I'm kind of lost here. What am I supposed to do at this point?
> 
> I can view the console.log for SlingPlayer and extract the hashed password.
> 
> I try to use the resuscitate guide, but get stuck after making a new device and trying to select SLINGBOX SETTINGS. It never gets passed step 8.
> 
> Gerry says that Slinger will process hashed passwords and v4 of Slinger will also provide the password in plain-text.
> 
> But I cannot get the device passed the wizard or registered to my Slingbox account.
> 
> Any advice?


I had a similar thing going on here: https://github.com/GerryDazoo/Slinger/discussions/598#discussioncomment-6180431

**Reply by GR0DZ1LLA** [reply]
Ok I read this and I have an IP from my DHCP server for the SB.  How would I prove or disprove if the SB has been bricked?

It shows in my list and shows as YELLOW and ready to Setup, but I cannot move forward.

![image](https://github.com/GerryDazoo/Slinger/assets/115742631/0da0718c-3b62-4c4a-9451-aa4cee5df25a)
